src/main/java/com/brightscout/ews/service/impl/CalendarServiceImpl.java

```diff
@@ -115,6 +126,7 @@ public ResponseEntity<Calendar> getCalendar(ExchangeService service, String cale
 	@Override
 	public ResponseEntity<MeetingTimeSuggestionResults> findMeetingTimes(ExchangeService service, String organizerEmail,
 			FindMeetingTimesParameters findMeetingTimes) throws InternalServerException {
+		logger.debug("Finding Meeting times for organizer: {}", organizerEmail);
```

Comment:
I think we find meeting times for all the attendees, not just the organizer. Also, there's no need here for logging.